lib/p2p/Pool.ts

```diff
@@ -290,6 +296,25 @@ class Pool extends EventEmitter {
     }
   }
 
+  /**
+   * Get node info
+   * @param nodePubKey to get reputation events
```

Comment:
Can you change this to "The node pub key of the node for which to get reputation information"?